lib/spotlight/dor/indexer.rb

```diff
@@ -141,6 +141,7 @@ def stacks_iiif_url(bare_druid, file_name)
       included do
         before_index :add_donor_tags
         before_index :add_folder_name
+        before_index :add_document_subtype
```

Comment:
maybe make these and the methods below alphabetical?  Trying to make it easier to find code is good.